pom.xml

```diff
@@ -79,6 +79,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <cal-branding-expanded>Codice Alliance</cal-branding-expanded>
```

Comment:
Do we have any general guidance on when to use`branding- expanded` vs just `branding`?